cert_manager/certbot.py

```diff
@@ -34,7 +36,15 @@ def request_cert(self, domains):
         for domain in domains:
             command += ["--domain", domain]
         result = subprocess.run(command, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-        result.check_returncode()
+        try:
+            result.check_returncode()
+        except subprocess.CalledProcessError as e:
+            if self.failure_alert_email:
+                message = "Subject: {subject}\n\n{stderr}".format(
```

Comment:
There should be a unit test for this and writing one will be easier if this code is split into a separate function.